config/locales/application_states.yml

```diff
@@ -158,6 +158,11 @@ en:
       description: |
         The provider has rejected the candidate’s application.
 
+    offer_withdrawn:
+      name: Rejected
```

Comment:
Should this be "Offer withdrawn"?

Reply:
Done - also mentioned the difference in API.